src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -175,8 +175,8 @@ export function EmbeddedPositionSearchPanel({
                         border: `3px solid ${theme.vars.palette['neutral']?.softActiveBg}`,
                         position: 'absolute',
                         px: 1/2,
-                        bottom: '1.5rem',
-                        left: 3,
+                        bottom: '3.5rem',
```

Comment:
Oh I didn't notice this before - I made a comment about this in your ife PR. The bottom can be 2.5rem maybe: 1rem (same as left) + 1.5rem (height + y margins of mouse-readout line)